index.html

```diff
@@ -62,31 +68,43 @@
     return (toUTC(end) - toUTC(start)) / msDay;
 }
 
+function addDays(date, days) {

```

Comment:
Either you return a new (constructed) date with the added days or you modify the input parameter. Doing both is a bit confusing.